### PR TITLE
Adds SLO to frontend

### LINKED ISSUE #3745: Handle Single Logout
I've added code on in vets-api to support the flow described [here](https://github.com/department-of-veterans-affairs/platform-team/blob/master/identity/logout.png). We need to have the client use that flow in order to log out of the IDP.

### PR DESCRIPTION
**Pull request by crwallace**
- Adds the ability for a user to log out of ID.me and their api session.

closes #3745

### CODE REVIEW

File: src/js/login/components/SignInProfileButton.jsx [hunk 1]
```diff
@@ -25,7 +13,7 @@ class SignInProfileButton extends React.Component {
         <span>
           <span>Hello, {this.props.profile.email}</span><span className="signin-spacer">|</span>
           <a href="/profile">Profile</a><span className="signin-spacer">|</span>
-          <a href="#" onClick={this.clearUserToken}>Sign Out</a>
+          <a ref="#" onClick={this.props.onUserLogout}>Sign Out</a>
```
Comment (bshyong) [inline src/js/login/components/SignInProfileButton.jsx]: Did you mean `href` here instead of `ref`?

File: content/pages/logout.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+<!-- Maintenance Page Start -->
+
+<div class="main home" role="main">
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <div style="padding: 2em 0;">
+        <h3>I am the logout <page class=""></page></h3>
```
Comment (bshyong) [inline content/pages/logout.md]: what is this `<page>` for?

**Review by bshyong — APPROVED** [review]
left some minor questions but LGTM

File: content/pages/logout.md (hunk 2 above)
Comment (crwallace) [inline content/pages/logout.md]: hmmm I have no idea what `<page>` is. I will remove this.

**Pull request merged**